src/Core/Logging/Log.cc

```diff
@@ -94,8 +94,8 @@ namespace
   protected:
     void sink_it_(const spdlog::details::log_msg& msg) override
     {
-		  fmt::memory_buffer formatted;
-		  sink::formatter_->format(msg, formatted);
+		  spdlog::memory_buf_t formatted;
```

Comment:
I get some build errors with this:
```
/Users/jess/software/SCIRun_testing/src/Core/Logging/Log.cc:97:26: error: use of undeclared identifier 'formatted'; did you
      mean 'fmt::formatter'?
                  spdlog::memory_buf_t formatted;
                                       ^~~~~~~~~
                                       fmt::formatter
/Users/jess/software/SCIRun_testing/bin/Externals/Source/SpdLog_external/include/spdlog/fmt/bundled/core.h:491:8: note: 
      'fmt::formatter' declared here
struct formatter {
       ^
/Users/jess/software/SCIRun_testing/src/Core/Logging/Log.cc:97:5: error: no type named 'memory_buf_t' in namespace 'spdlog';
      did you mean 'fmt::memory_buffer'?
                  spdlog::memory_buf_t formatted;
                  ^~~~~~~~~~~~~~~~~~~~
                  fmt::memory_buffer
/Users/jess/software/SCIRun_testing/bin/Externals/Source/SpdLog_external/include/spdlog/fmt/bundled/format.h:553:35: note: 
      'fmt::memory_buffer' declared here
typedef basic_memory_buffer<char> memory_buffer;
```

Reply:
With two externals updated, you'll need to do a fresh Superbuild. Try blowing away the whole directory.